Library/Homebrew/cask/cask.rb

```diff
@@ -133,7 +133,7 @@ def to_h
         end,
         "caveats"        => caveats,
         "depends_on"     => depends_on,
-        "conflicts_with" => conflicts_with.to_a,
+        "conflicts_with" => conflicts_with.to_h,
```

Comment:
Does it work if you leave off `.to_h` altogether? Neither of the other values call this explicitly.

Reply:
It tries to produce the object string representation like this ```"conflicts_with":"#<Cask::DSL::ConflictsWith:0x00007f94700138d8>"```

Reply:
Ah, `depends_on` works without it because it subclasses `DelegateClass(Hash)`.